### ISSUE TITLE
Firmware update error

### ISSUE THREAD

**Issue by fabiodelgadopereira** [lesserkuma/FlashGBX#40]
I have this problem:
```
FlashGBX v3.13 by Lesserkuma
https://github.com/lesserkuma/FlashGBX

You are using the latest version of FlashGBX.

Connected to GBxCart RW v1.1/v1.2 – Firmware R30 (COM3)
For help please visit the insideGadgets Discord: https://gbxcart.com/discord

Now running in Legacy Mode.
This version of FlashGBX has been tested with GBxCart RW v1.3 and v1.4.
Other revisions may not be fully compatible.

NOTE: Update your GBxCart RW firmware to version L1 or higher for a better transfer rate with this cartridge.
```
![image](https://user-images.githubusercontent.com/38516312/172077778-fb553f13-b01b-4c30-8307-2fb9e8bafef9.png)

**Comment by lesserkuma** [comment]
Can you try if the official firmware updater one works?
https://shop.insidegadgets.com/wp-content/uploads/2018/05/GBxCart_RW_Firmware_Update_29Apr21.zip

Also it seems like with your v1.1/v1.2 device, you’re already on firmware version R30 which should be the latest. Unfortunately, the L1 firmware is only available for the v1.3 and v1.4 hardware revisions.

**Comment by fabiodelgadopereira** [comment]
Thanks!
